assets/src/edit-story/components/devTools/devTools.js

```diff
@@ -0,0 +1,210 @@
+            newElement.resource.poster =
+              'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAIAAAACCAYAAABytg0kAAAAGElEQVQYV2NkYGD4////fwZGEMnIyMgAAEvqB/6wfXLSAAAAAElFTkSuQmCC';
+
+            const videoEl = document.getElementById(`video-${newElement.id}`);
+            if (videoEl) {
+              videoEl.load();
+            }
+          }
+          if (element.type === 'image') {
+            newElement.resource.mimeType = 'image/png';
+            newElement.resource.src =
+              'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAIAAAACCAYAAABytg0kAAAAGElEQVQYV2NkYGD4////fwZGEMnIyMgAAEvqB/6wfXLSAAAAAElFTkSuQmCC';
+          }
+        }
+        return newElement;
+      }),
+    })),
+  };
+};
+
+function DevTools() {
+  const [isVisible, setIsVisible] = useState(false);
+  const [isBase64, setIsBase64] = useState(false);
+  const [isDummyResources, setIsDummyResources] = useState(false);
+  const { showSnackbar } = useSnackbar();
+  const textareaRef = useRef();
+  const {
+    state: { reducerState },
+    actions: { restore },
+  } = useStory();
+  // This will not work here
+  // const {
+  //   state, // stale data
+  //   internal: { restore }, // will not update
+  // } = useStoryReducer();
+
+  const {
+    pages,
+    current,
+    selection,
+    story: { stylePresets },
+  } = reducerState;
+  const reducerStateSlice = {
+    current,
+    selection,
+    story: { stylePresets },
+    pages,
+  };
+  const storyData = isDummyResources
```

Comment:
When would we not want `isDummyResources`?

Reply:
When you want to show original URLs (for debugging purposes) or share it on the same env (like staging - so no need to replace resources, I know that you can send the story URL, but still).

Reply:
Sounds good.